src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class StorageTest {
+    private final static String TEST_DATA_FOLDER_PATH = "./dataTest";
```

Comment:
I think you shouldn't use `private static final`, but `private final static`: https://google.github.io/styleguide/javaguide.html